config/packages/monolog.yaml

```diff
@@ -0,0 +1,65 @@
+monolog:
+  channels:
+    - deprecation # Deprecations are logged in the dedicated "deprecation" channel when it exists
+
+when@dev:
+  monolog:
+    handlers:
+      main:
+        type: rotating_file
+        path: "%kernel.logs_dir%/%kernel.environment%.log"
+        # Or use: "debug" instead of "info" for more log (debug) messages
+        level: info
+        channels: ["!event"]
+        # Enable full stacktrace, set this to false, to disable stacktraces
+        include_stacktraces: true
+        max_files: 10
+      # uncomment to get logging in your browser
+      # you may have to allow bigger header sizes in your Web server configuration
+      #firephp:
+      #    type: firephp
+      #    level: info
+      #chromephp:
+      #    type: chromephp
+      #    level: info
+      console:
+        type: console
+        process_psr_3_messages: false
+        channels: ["!event", "!doctrine", "!console"]
+
+when@test:
+  monolog:
+    handlers:
+      main:
+        type: fingers_crossed
+        action_level: error
+        handler: nested
+        excluded_http_codes: [404, 405]
+        channels: ["!event"]
+      nested:
+        type: stream
+        path: "%kernel.logs_dir%/%kernel.environment%.log"
+        level: debug
+
+when@prod:
+  monolog:
+    handlers:
+      main:
+        type: fingers_crossed
+        action_level: error
+        handler: nested
+        buffer_size: 50 # How many messages should be saved? Prevent memory leaks
```

Comment:
maybe add `excluded_http_codes: [404, 405]` config here too, like the old prod log